components/Dialogs/src/MDCAlertController.m

```diff
@@ -329,6 +330,14 @@ - (void)setTitleAlignment:(NSTextAlignment)titleAlignment {
   }
 }
 
+// todo: test alignment getter - after changing the alert.view.messageAlignment
```

Comment:
Please file an Issue to track this TODO and link to it here in the form `TODO(<link to Issue>): ...`.